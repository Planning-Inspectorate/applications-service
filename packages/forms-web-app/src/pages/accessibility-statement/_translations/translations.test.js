const enAccessibilityStatementTranslations = require('./en.json');
const cyAccessibilityStatementTranslations = require('./cy.json');

describe('pages/accessibility-statement/_translations', () => {
	it('should return the correct English translations', () => {
		expect(enAccessibilityStatementTranslations).toMatchSnapshot();
	});

	it('should return the correct Welsh translations', () => {
		expect(cyAccessibilityStatementTranslations).toMatchSnapshot();
	});

	it('should return the same keys for the English and Welsh translations', () => {
		expect(enAccessibilityStatementTranslations).toHaveSameKeys(
			cyAccessibilityStatementTranslations
		);
	});
});
