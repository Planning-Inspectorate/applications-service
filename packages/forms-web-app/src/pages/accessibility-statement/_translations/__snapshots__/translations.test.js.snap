// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/accessibility-statement/_translations should return the correct English translations 1`] = `
{
  "heading1": "Accessibility statement for national infrastructure projects",
  "heading10": "Non-compliance with the accessibility regulations",
  "heading11": "PDFs and non-HTML documents",
  "heading12": "Content that's not within the scope of the accessibility regulations",
  "heading13": "PDFs and other documents",
  "heading14": "Preparation of this accessibility statement ",
  "heading2": "Using this service",
  "heading3": "How accessible this website is",
  "heading4": "Feedback and contact information",
  "heading5": "Reporting accessibility problems with this service",
  "heading6": "Enforcement procedure",
  "heading7": "Technical information about this website's accessibility",
  "heading8": "Compliance status",
  "heading9": "Non-accessible content",
  "listItem1": "change colours, contrast levels and fonts",
  "listItem10": "Email:",
  "listItem11": "Call:",
  "listItem12": "It has been observed that if a user is navigating the page with Dragon Naturally Speaking enabled, they are unable to expand the additional information on the Project Information details components. This does not comply with WCAG Level A 2.1: 4.1.2 Name, Role, Value. This issue requires a fix in Dragon Naturally Speaking. As an alternative, users can use MouseGrid, a feature in the Windows Speech Recognition app.",
  "listItem13": "If a user is navigating the page with iOS VoiceOver enabled, the user is not notified that there is additional information which has not been expanded on the screen. Because of this, the user would not know that there is more information within the Project Information components to be heard. If the user does click on the section to expand it, there is also no announcement to say this has happened. This does not comply with WCAG Level A 2.1: Name, Role, Value.",
  "listItem14": "For users of assistive technology the ability to upload files with Dragon Naturally Speaking software fails. This does not comply with WCAG Level A 2.1: 4.1.2 Name, Role, Value.",
  "listItem15": "some of our PDFs are not accessible as they are not tagged properly",
  "listItem16": "some PDFs, Word and Excel documents fail on the perceivable and operable WCAG2 criteria",
  "listItem17": "a few of our documents have diagrams. These images do not have a text alternative.",
  "listItem18": "some of the documents we publish are produced by third parties. We are not always able to make these fully compliant",
  "listItem2": "zoom in up to 300% without the text spilling off the screen",
  "listItem3": "navigate most of the website using just a keyboard",
  "listItem4": "navigate most of the website using speech recognition software",
  "listItem5": "listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)",
  "listItem6": "some pages and document attachments are not written in plain English",
  "listItem7": "many documents are in PDF format and are not accessible",
  "listItem8": "there may be some links which do not provide enough context",
  "listItem9": "people using assistive technology may find some parts of the site difficult to navigate",
  "paragraph1": "This accessibility statement applies to",
  "paragraph10": "We're always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we're not meeting accessibility requirements, email us at: ",
  "paragraph11": "If you contact us with a complaint and you're not happy with our response {{-link}}.",
  "paragraph11LinkText": "contact the Equality Advisory and Support Service (EASS)",
  "paragraph12": "The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations').",
  "paragraph13": "The Planning Inspectorate is committed to making its websites accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.",
  "paragraph14": "This website is partially compliant with the Web Content Accessibility Guidelines version 2.1 AA standard. The non-compliances are listed below.",
  "paragraph15": "The content listed below is non-accessible for the following reasons.",
  "paragraph16": "Many documents are not accessible in a number of ways including missing text alternatives and missing document structure. For example, they may not be structured so they're accessible to a screen reader. This does not meet WCAG 2.1 success criterion 4.1.2. Some of our PDFs and Word documents are essential to providing our services. For example, we have PDFs with information on how users can access our services, and forms published as Word documents.",
  "paragraph17": "View the {{-link}} of the organisation that published the document to report any problems or request documents in an alternative format. If more than one organisation is listed, view the accessible document policy of the first.",
  "paragraph17LinkText": "accessible document policy",
  "paragraph18": "Where possible, we try to fix these as soon as we can.",
  "paragraph19": "Some of our PDFs and Word documents are essential to providing our services. For example, we have PDFs with information on how users can access our services, and forms published as Word documents.",
  "paragraph2": "This website is run by the Planning Inspectorate. We want as many people as possible to be able to use this website. For example, that means you should be able to:",
  "paragraph20": "The accessibility regulations {{-link}} if they're not essential to providing our services.",
  "paragraph20LinkText": "do not require us to fix PDFs or other documents published before 23 September 2018",
  "paragraph21": "An accessibility report of the service took place in October 2023. This statement was prepared and last reviewed on the 8th of November 2023. This service is tested at least monthly by the team that maintains it, and periodically by external bodies.",
  "paragraph3": "We've also made the text as simple as possible to understand.",
  "paragraph4": "{{-link}} has advice on making your device easier to use if you have a disability.",
  "paragraph4LinkText": "AbilityNet",
  "paragraph5": "This service is partially compliant with the {{-link}} AA standard.",
  "paragraph5LinkText": "Web Content Accessibility Guidelines version 2.1",
  "paragraph6": "We know some parts of this website are not fully accessible:",
  "paragraph7": "The compliance status section below gives more detail on the areas of the site that are not fully accessible.",
  "paragraph8": "If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:",
  "paragraph9": "We'll consider your request and get back to you in 10 working days.",
}
`;

exports[`pages/accessibility-statement/_translations should return the correct Welsh translations 1`] = `
{
  "heading1": "Datganiad hygyrchedd ar gyfer prosiectau seilwaith cenedlaethol",
  "heading10": "Diffyg cydymffurfio â'r rheoliadau hygyrchedd",
  "heading11": "Dogfennau PDF a dogfennau nad ydynt yn HTML",
  "heading12": "Cynnwys nad yw o fewn cwmpas y rheoliadau hygyrchedd",
  "heading13": "Dogfennau PDF a dogfennau eraill",
  "heading14": "Paratoi'r datganiad hygyrchedd hwn",
  "heading2": "Defnyddio'r gwasanaeth hwn",
  "heading3": "Pa mor hygyrch yw'r wefan hon",
  "heading4": "Adborth a gwybodaeth gyswllt",
  "heading5": "Adrodd am broblemau hygyrchedd â'r gwasanaeth hwn",
  "heading6": "Gweithdrefn orfodi",
  "heading7": "Gwybodaeth dechnegol am hygyrchedd y wefan hon",
  "heading8": "Statws cydymffurfio",
  "heading9": "Cynnwys nad yw'n hygyrch",
  "listItem1": "newid lliwiau, lefelau cyferbynnedd a ffontiau",
  "listItem10": "Anfonwch neges e-bost at:",
  "listItem11": "Ffoniwch:",
  "listItem12": "Arsylwyd os yw defnyddiwr yn llywio'r dudalen gan ddefnyddio meddalwedd Dragon Naturally Speaking, nid yw'n gallu ehangu'r wybodaeth ychwanegol yn yr elfennau manylion Gwybodaeth am y Prosiect. Nid yw hyn yn cydymffurfio â Lefel A: 4.1.2 Enw, Rôl, Gwerth y Canllawiau Hygyrchedd Cynnwys Gwe (WCAG) 2.1. Mae angen i'r mater hwn gael ei ddatrys yn Dragon Naturally Speaking. Yn lle hynny, gall defnyddwyr ddefnyddio MouseGrid, sef nodwedd yn ap Adnabod Llais Windows.",
  "listItem13": "Os yw defnyddiwr yn llywio'r dudalen gan ddefnyddio iOS VoiceOver, ni roddir gwybod i'r defnyddiwr fod gwybodaeth ychwanegol nad yw wedi cael ei hehangu ar y sgrin. Oherwydd hyn, ni fyddai'r defnyddiwr yn gwybod bod mwy o wybodaeth i'w chlywed o fewn yr elfennau Gwybodaeth am y Prosiect. Os yw'r defnyddiwr yn clicio ar yr adran i'w hehangu, nid oes cyhoeddiad i ddweud bod hyn wedi digwydd. Nid yw hyn yn cydymffurfio â Lefel A: Enw, Rôl, Gwerth WCAG. 2.1.",
  "listItem14": "I'r rhai sy'n defnyddio technoleg gynorthwyol, mae'r gallu i lanlwytho ffeiliau gan ddefnyddio meddalwedd Dragon Naturally Speaking yn methu. Nid yw hyn yn cydymffurfio â Lefel A: 4.1.2 Enw, Rôl, Gwerth WCAG 2.1.",
  "listItem15": "nid yw rhai o'n dogfennau PDF yn hygyrch gan nad ydynt wedi'u tagio'n briodol",
  "listItem16": "mae rhai dogfennau PDF, Word ac Excel yn methu maen prawf canfyddadwy ac ymarferol WCAG2",
  "listItem17": "mae rhai o'n dogfennau'n cynnwys diagramau. Nid oes testun amgen ar gyfer y delweddau hyn",
  "listItem18": "mae rhai o'r dogfennau a gyhoeddwn yn cael eu cynhyrchu gan drydydd parti. Nid ydym bob amser yn gallu sicrhau bod y rhain yn cydymffurfio'n llwyr",
  "listItem2": "chwyddo i mewn hyd at 300% heb i'r testun ddiflannu oddi ar y sgrin",
  "listItem3": "llywio'r rhan fwyaf o'r wefan gan ddefnyddio bysellfwrdd yn unig",
  "listItem4": "llywio'r rhan fwyaf o'r wefan gan ddefnyddio meddalwedd adnabod llais",
  "listItem5": "gwrando ar y rhan fwyaf o'r wefan gan ddefnyddio rhaglen darllen sgrin (gan gynnwys fersiynau diweddaraf JAWS, NVDA a VoiceOver)",
  "listItem6": "nid yw rhai tudalennau a dogfennau atodedig wedi'u hysgrifennu mewn iaith glir",
  "listItem7": "mae llawer o ddogfennau mewn fformat PDF ac nid ydynt yn hygyrch",
  "listItem8": "fe allai fod rhai dolenni nad ydynt yn rhoi digon o gyd-destun",
  "listItem9": "fe allai rhai rhannau o'r safle fod yn anodd eu llywio i bobl sy'n defnyddio technoleg gynorthwyol",
  "paragraph1": "Mae'r datganiad hygyrchedd hwn yn berthnasol i",
  "paragraph10": "Rydym bob amser yn ceisio gwella hygyrchedd y wefan hon. Os dewch ar draws unrhyw broblemau nad ydynt wedi'u rhestru ar y dudalen hon neu os ydych yn credu nad ydym yn bodloni ein gofynion hygyrchedd, anfonwch neges e-bost atom:",
  "paragraph11": "Os byddwch yn cysylltu â ni gyda chwyn ac yn anfodlon ar ein hymateb, {{-link}}.",
  "paragraph11LinkText": "cysylltwch â'r Gwasanaeth Cynghori a Chymorth Cydraddoldeb (EASS)",
  "paragraph12": "Mae'r Comisiwn Cydraddoldeb a Hawliau Dynol (EHRC) yn gyfrifol am orfodi Rheoliadau Hygyrchedd Cyrff Sector Cyhoeddus (Gwefannau a Chymwysiadau Symudol) (Rhif 2) 2018 (y 'rheoliadau hygyrchedd').",
  "paragraph13": "Mae'r Arolygiaeth Gynllunio wedi ymrwymo i wneud ei gwefannau'n hygyrch, yn unol â Rheoliadau Hygyrchedd Cyrff Sector Cyhoeddus (Gwefannau a Chymwysiadau Symudol) (Rhif 2) 2018.",
  "paragraph14": "Mae'r wefan hon yn cydymffurfio'n rhannol â safon AA y Canllawiau Hygyrchedd Cynnwys Gwe fersiwn 2.1. Rhestrir yr achosion o ddiffyg cydymffurfio isod.",
  "paragraph15": "Mae'r cynnwys a restrir isod yn anhygyrch am y rhesymau canlynol.",
  "paragraph16": "Nid yw llawer o ddogfennau'n hygyrch mewn nifer o ffyrdd, gan gynnwys testun amgen ar goll a strwythur dogfen ar goll. Er enghraifft, efallai nad ydynt wedi'u strwythuro fel eu bod yn hygyrch i raglen darllen sgrin. Nid yw hyn yn bodloni maen prawf llwyddo 4.1.2 WCAG 2.1. Mae rhai o'n dogfennau PDF a Word yn hanfodol i ddarparu ein gwasanaethau. Er enghraifft, mae gennym ddogfennau PDF sy'n cynnwys gwybodaeth am sut gall defnyddwyr gael at ein gwasanaethau, a ffurflenni wedi'u cyhoeddi fel dogfennau Word.",
  "paragraph17": "Darllenwch {{-link}} y sefydliad a gyhoeddodd y ddogfen i adrodd am unrhyw broblemau neu ofyn am ddogfennau mewn fformat amgen. Os oes mwy nag un sefydliad wedi'i restru, darllenwch bolisi dogfen hygyrch yr un cyntaf.",
  "paragraph17LinkText": "bolisi dogfen hygyrch",
  "paragraph18": "Ceisiwn ddatrys y rhain cyn gynted ag y gallwn, lle y bo'n bosibl.",
  "paragraph19": "Mae rhai o'n dogfennau PDF a Word yn hanfodol i ddarparu ein gwasanaethau. Er enghraifft, mae gennym ddogfennau PDF sy'n cynnwys gwybodaeth am sut gall defnyddwyr gael at ein gwasanaethau, a ffurflenni wedi'u cyhoeddi fel dogfennau Word.",
  "paragraph2": "Mae'r wefan hon yn cael ei chynnal gan yr Arolygiaeth Gynllunio. Rydym eisiau i gynifer o bobl â phosibl ddefnyddio'r wefan hon. Er enghraifft, mae hyn yn golygu y dylech allu:",
  "paragraph20": "{{-link}} os nad ydynt yn hanfodol i ddarparu ein gwasanaethau.",
  "paragraph20LinkText": "Nid yw'r rheoliadau hygyrchedd yn mynnu ein bod yn cywiro dogfennau PDF na dogfennau eraill a gyhoeddwyd cyn 23 Medi 2018",
  "paragraph21": "Lluniwyd adroddiad hygyrchedd ar y gwasanaeth ym mis Hydref 2023. Paratowyd y datganiad hwn ac fe'i hadolygwyd ddiwethaf ar 8 Tachwedd 2023. Mae'r gwasanaeth hwn yn cael ei brofi'n fisol o leiaf gan y tîm sy'n ei gynnal, ac yn gyfnodol gan gyrff allanol.",
  "paragraph3": "Rydym hefyd wedi gwneud y testun mor hawdd â phosibl ei ddeall.",
  "paragraph4": "Mae {{-link}} yn cynnig cyngor ar wneud eich dyfais yn haws ei defnyddio os oes gennych anabledd.",
  "paragraph4LinkText": "AbilityNet",
  "paragraph5": "Mae'r gwasanaeth hwn yn cydymffurfio'n rhannol â safon AA y {{-link}}.",
  "paragraph5LinkText": "Canllawiau Hygyrchedd Cynnwys Gwe fersiwn 2.1.",
  "paragraph6": "Rydym yn gwybod nad yw rhai rhannau o'r wefan hon yn gwbl hygyrch:",
  "paragraph7": "Mae'r adran isod ar statws cydymffurfio yn rhoi mwy o fanylion am y rhannau o'r safle nad ydynt yn gwbl hygyrch.",
  "paragraph8": "Os oes arnoch angen gwybodaeth ar y wefan hon mewn fformat gwahanol fel PDF hygyrch, print bras, fersiwn hawdd ei darllen, recordiad sain neu braille:",
  "paragraph9": "Byddwn yn ystyried eich cais ac yn cysylltu â chi ymhen 10 niwrnod gwaith.",
}
`;
